src/main/java/com/sandogh/sandogh/users/UserDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.sandogh.sandogh.users;
```

Comment:
This class is not used inside the project. Remove the class if you don't need it.